Schutzfile

```diff
@@ -68,6 +68,27 @@
       }
     ]
   },
+  "fedora-43": {
+    "repos": [
+      {
+        "file": "/etc/yum.repos.d/fedora.repo",
+        "x86_64": [
+          {
+            "title": "fedora",
+            "name": "fedora",
+            "baseurl": "https://rpmrepo.osbuild.org/v2/mirror/public/f43/f43-x86_64-branched-20250901"
```

Comment:
nitpick: other repo snapshots in this file are already from `20251001`, but that will get fixed by the next scheduled snapshot.